### DISCUSSION TITLE
Portierung von CCU2 mit JP-HB-Devices-addon

### DISCUSSION THREAD

**Discussion by akula3** [OpenCCU/OpenCCU#1076] [🇩🇪 Fragen&Antworten] [answered]
Hallo,
ich benutze [JP-HB-Devices-addon](https://github.com/jp112sdl/JP-HB-Devices-addon) und möchte von CCU2 auf Raspberrymatic umsteigen.
In der Wiki steht:

> Nun unter Einstellungen->Systemsteuerung->Zusatzsoftware die Liste der Zusatzpakete/CCU-Addons durchgehen und grundsätzlich ALLE deinstallieren, jedoch die folgenden Addons hiervon ausnehmen da ansonsten deren Konfiguration mit gelöscht wird (diese können später unter Punkt 10 dann wieder "drüber" installiert werden):

    CUxD
    HB-TM-Devices-AddOn

Da die Addon Funktionalitäten von TomMajor und Jérôme ähnlich sind, meine Frage:
Muss das JP-HB-Devices-addon ähnlich wie HB-TM-Devices-AddOn **_nicht_** deinstalliert werden?

**Comment by jens-maus** [comment]
Na die Frage gebe ich gerne an @jp112sdl weiter :)

**Comment by jp112sdl** [comment, marked as answer]
Hi @akula3 

Ja korrekt, das JP-HB-Devices-addon **darf** sogar **nicht deinstalliert werden**, weil sonst beim ersten Hochfahren der RaspberryMatic die Geräte alle "rausfliegen", da sie unbekannt sind.

Also Addon drauf lassen, Backup machen, bei Raspberrymatic einspielen und alles läuft so weiter wie bisher

**Reply by akula3** [reply]
Danke, Jérôme!

**Reply by jens-maus** [reply]
Danke auch von mir für die Info. Hab es nun so auch in der Doku entsprechend ergänzt!

**Reply by jp112sdl** [reply]
> Danke auch von mir für die Info. Hab es nun so auch in der Doku entsprechend ergänzt!

Ich weiß gar nicht, ob es überhaupt (noch) notwendig ist, Addons zu deinstallieren, bzw. wo es noch eine CPU-abhängige Installation gibt (außer RedMatic, was ohnehin nicht auf der CCU2 läuft)...
Außer bei CUxD braucht diese "Spezialbehandlung" doch eigentlich niemand mehr oder?
